aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -1045,8 +1049,14 @@ std::tuple<at::Tensor, at::Tensor, Tensor, Tensor> _efficient_attention_forward(
 
       // assign strides for bias, viewed as
       // (batch_sz, n_heads, n_queries, n_keys)
-      const at::Tensor bias_4d_view =
-          get_bias_4d_view(*bias, B, num_heads, M, N);
+      // We make sure to expand prior to calling the kernel
```

Comment:
Are there any checks in place at kernel level that'll throw an exception if the bias' shape isn't a multiple of 16?

Reply:
Yeah, this torch_check will pop: https://github.com/pytorch/pytorch/blob/14304afd76a6e8c383fe833d1f28122d45ae99eb/aten/src/ATen/native/transformers/cuda/mem_eff_attention/kernel_forward.h#L583